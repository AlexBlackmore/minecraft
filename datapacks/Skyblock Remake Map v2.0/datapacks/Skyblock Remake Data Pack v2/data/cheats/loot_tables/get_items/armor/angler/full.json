{
  "pools": [
    {
      "rolls": 1,
      "entries": [
        {
          "type": "item",
          "name": "minecraft:diamond_helmet",
          "functions": [
            {
              "function": "minecraft:set_nbt",
              "tag": "{Enchantments:[{}],Base:{Type:\"HELMET\",Rarity:\"COMMON\",RarityColor:'{\"text\":\"\",\"color\":\"white\"}',AnvilUses:0,RepairCost:0,Enchantments:[],Reforge:\"\",Defense:15,BaseDefense:15,SCChance:1,BaseSCChance:1,Abilities:[{Name:\"Depth Champion\",Activation:\"\",Type:\"Full Set Bonus\"},{Name:\"DeepnessWithin\",Activation:\"\",Type:\"Full Set Bonus\"}]},HideFlags:63,Unbreakable:1b,Description:['[{\"text\":\"Full Set Bonus: \",\"color\":\"gold\",\"italic\":false},{\"text\":\"Depth Champion\",\"color\":\"gold\",\"italic\":false},{\"text\":\" \",\"color\":\"yellow\",\"italic\":false,\"bold\":true}]','[{\"text\":\"Take \",\"color\":\"gray\",\"italic\":false},{\"text\":\"-30% \",\"color\":\"green\",\"italic\":false},{\"text\":\"damage from Sea Creatures. \",\"color\":\"gray\",\"italic\":false}]','[{\"text\":\"Increase their spawn rate by \",\"color\":\"gray\",\"italic\":false},{\"text\":\"4%\",\"color\":\"yellow\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"Full Set Bonus: \",\"color\":\"gold\",\"italic\":false},{\"text\":\"Deepness Within\",\"color\":\"gold\",\"italic\":false},{\"text\":\" \",\"color\":\"yellow\",\"italic\":false,\"bold\":true}]','[{\"text\":\"Gain \",\"color\":\"gray\",\"italic\":false},{\"text\":\"10❤ \",\"color\":\"red\",\"italic\":false},{\"text\":\"per Fishing level. \",\"color\":\"gray\",\"italic\":false}]','[{\"text\":\"Fishing: \",\"color\":\"gray\",\"italic\":false},{\"text\":\"Level I\",\"color\":\"yellow\",\"italic\":false}]','[{\"text\":\"\"}]'],display:{Name:'[{\"text\":\"Angler Helmet\",\"color\":\"white\",\"italic\":false}]',Lore:['[{\"text\":\"Sea Creature Chance: \",\"color\":\"gray\",\"italic\":false},{\"text\":\"+1%\",\"color\":\"red\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"Defense: \",\"color\":\"gray\",\"italic\":false},{\"text\":\"+15\",\"color\":\"green\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"Full Set Bonus: \",\"color\":\"gold\",\"italic\":false},{\"text\":\"Depth Champion\",\"color\":\"gold\",\"italic\":false},{\"text\":\" \",\"color\":\"yellow\",\"italic\":false,\"bold\":true}]','[{\"text\":\"Take \",\"color\":\"gray\",\"italic\":false},{\"text\":\"-30% \",\"color\":\"green\",\"italic\":false},{\"text\":\"damage from Sea Creatures. \",\"color\":\"gray\",\"italic\":false}]','[{\"text\":\"Increase their spawn rate by \",\"color\":\"gray\",\"italic\":false},{\"text\":\"4%\",\"color\":\"yellow\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"Full Set Bonus: \",\"color\":\"gold\",\"italic\":false},{\"text\":\"Deepness Within\",\"color\":\"gold\",\"italic\":false},{\"text\":\" \",\"color\":\"yellow\",\"italic\":false,\"bold\":true}]','[{\"text\":\"Gain \",\"color\":\"gray\",\"italic\":false},{\"text\":\"10❤ \",\"color\":\"red\",\"italic\":false},{\"text\":\"per Fishing level. \",\"color\":\"gray\",\"italic\":false}]','[{\"text\":\"Fishing: \",\"color\":\"gray\",\"italic\":false},{\"text\":\"Level I\",\"color\":\"yellow\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"This item can be reforged!\",\"color\":\"dark_gray\",\"italic\":false}]','[{\"text\":\"COMMON HELMET\",\"color\":\"white\",\"italic\":false,\"bold\":true}]']}}"
            },
            {
                "function": "minecraft:set_attributes",
                "modifiers": [
                  {
                    "attribute": "minecraft:generic.armor",
                    "name": "",
                    "amount": 0,
                    "operation": "multiply_total",
                    "slot": ["feet","legs","chest","head"]
                  }
                ]
            }
          ]
        }
      ]
    },
    {
      "rolls": 1,
      "entries": [
        {
          "type": "item",
          "name": "minecraft:leather_chestplate",
          "functions": [
            {
              "function": "minecraft:set_nbt",
              "tag": "{Base:{Type:\"CHESTPLATE\",Rarity:\"COMMON\",RarityColor:'{\"text\":\"\",\"color\":\"white\"}',AnvilUses:0,RepairCost:0,Enchantments:[],Reforge:\"\",Defense:40,BaseDefense:40,SCChance:1,BaseSCChance:1,Abilities:[{Name:\"Depth Champion\",Activation:\"\",Type:\"Full Set Bonus\"},{Name:\"DeepnessWithin\",Activation:\"\",Type:\"Full Set Bonus\"}]},HideFlags:63,Unbreakable:1b,Description:['[{\"text\":\"Full Set Bonus: \",\"color\":\"gold\",\"italic\":false},{\"text\":\"Depth Champion\",\"color\":\"gold\",\"italic\":false},{\"text\":\" \",\"color\":\"yellow\",\"italic\":false,\"bold\":true}]','[{\"text\":\"Take \",\"color\":\"gray\",\"italic\":false},{\"text\":\"-30% \",\"color\":\"green\",\"italic\":false},{\"text\":\"damage from Sea Creatures. \",\"color\":\"gray\",\"italic\":false}]','[{\"text\":\"Increase their spawn rate by \",\"color\":\"gray\",\"italic\":false},{\"text\":\"4%\",\"color\":\"yellow\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"Full Set Bonus: \",\"color\":\"gold\",\"italic\":false},{\"text\":\"Deepness Within\",\"color\":\"gold\",\"italic\":false},{\"text\":\" \",\"color\":\"yellow\",\"italic\":false,\"bold\":true}]','[{\"text\":\"Gain \",\"color\":\"gray\",\"italic\":false},{\"text\":\"10❤ \",\"color\":\"red\",\"italic\":false},{\"text\":\"per Fishing level. \",\"color\":\"gray\",\"italic\":false}]','[{\"text\":\"Fishing: \",\"color\":\"gray\",\"italic\":false},{\"text\":\"Level I\",\"color\":\"yellow\",\"italic\":false}]','[{\"text\":\"\"}]'],display:{Name:'[{\"text\":\"Angler Chestplate\",\"color\":\"white\",\"italic\":false}]',color:720975,Lore:['[{\"text\":\"Sea Creature Chance: \",\"color\":\"gray\",\"italic\":false},{\"text\":\"+1%\",\"color\":\"red\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"Defense: \",\"color\":\"gray\",\"italic\":false},{\"text\":\"+40\",\"color\":\"green\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"Full Set Bonus: \",\"color\":\"gold\",\"italic\":false},{\"text\":\"Depth Champion\",\"color\":\"gold\",\"italic\":false},{\"text\":\" \",\"color\":\"yellow\",\"italic\":false,\"bold\":true}]','[{\"text\":\"Take \",\"color\":\"gray\",\"italic\":false},{\"text\":\"-30% \",\"color\":\"green\",\"italic\":false},{\"text\":\"damage from Sea Creatures. \",\"color\":\"gray\",\"italic\":false}]','[{\"text\":\"Increase their spawn rate by \",\"color\":\"gray\",\"italic\":false},{\"text\":\"4%\",\"color\":\"yellow\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"Full Set Bonus: \",\"color\":\"gold\",\"italic\":false},{\"text\":\"Deepness Within\",\"color\":\"gold\",\"italic\":false},{\"text\":\" \",\"color\":\"yellow\",\"italic\":false,\"bold\":true}]','[{\"text\":\"Gain \",\"color\":\"gray\",\"italic\":false},{\"text\":\"10❤ \",\"color\":\"red\",\"italic\":false},{\"text\":\"per Fishing level. \",\"color\":\"gray\",\"italic\":false}]','[{\"text\":\"Fishing: \",\"color\":\"gray\",\"italic\":false},{\"text\":\"Level I\",\"color\":\"yellow\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"This item can be reforged!\",\"color\":\"dark_gray\",\"italic\":false}]','[{\"text\":\"COMMON CHESTPLATE\",\"color\":\"white\",\"italic\":false,\"bold\":true}]'],color:720975}}"
            },
            {
                "function": "minecraft:set_attributes",
                "modifiers": [
                  {
                    "attribute": "minecraft:generic.armor",
                    "name": "",
                    "amount": 0,
                    "operation": "multiply_total",
                    "slot": ["feet","legs","chest","head"]
                  }
                ]
            }
          ]
        }
      ]
    },
    {
      "rolls": 1,
      "entries": [
        {
          "type": "item",
          "name": "minecraft:leather_leggings",
          "functions": [
            {
              "function": "minecraft:set_nbt",
              "tag": "{Base:{Type:\"LEGGINGS\",Rarity:\"COMMON\",RarityColor:'{\"text\":\"\",\"color\":\"white\"}',AnvilUses:0,RepairCost:0,Enchantments:[],Reforge:\"\",Defense:25,BaseDefense:25,SCChance:1,BaseSCChance:1,Abilities:[{Name:\"Depth Champion\",Activation:\"\",Type:\"Full Set Bonus\"},{Name:\"DeepnessWithin\",Activation:\"\",Type:\"Full Set Bonus\"}]},HideFlags:63,Unbreakable:1b,Description:['[{\"text\":\"Full Set Bonus: \",\"color\":\"gold\",\"italic\":false},{\"text\":\"Depth Champion\",\"color\":\"gold\",\"italic\":false},{\"text\":\" \",\"color\":\"yellow\",\"italic\":false,\"bold\":true}]','[{\"text\":\"Take \",\"color\":\"gray\",\"italic\":false},{\"text\":\"-30% \",\"color\":\"green\",\"italic\":false},{\"text\":\"damage from Sea Creatures. \",\"color\":\"gray\",\"italic\":false}]','[{\"text\":\"Increase their spawn rate by \",\"color\":\"gray\",\"italic\":false},{\"text\":\"4%\",\"color\":\"yellow\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"Full Set Bonus: \",\"color\":\"gold\",\"italic\":false},{\"text\":\"Deepness Within\",\"color\":\"gold\",\"italic\":false},{\"text\":\" \",\"color\":\"yellow\",\"italic\":false,\"bold\":true}]','[{\"text\":\"Gain \",\"color\":\"gray\",\"italic\":false},{\"text\":\"10❤ \",\"color\":\"red\",\"italic\":false},{\"text\":\"per Fishing level. \",\"color\":\"gray\",\"italic\":false}]','[{\"text\":\"Fishing: \",\"color\":\"gray\",\"italic\":false},{\"text\":\"Level I\",\"color\":\"yellow\",\"italic\":false}]','[{\"text\":\"\"}]'],display:{Name:'[{\"text\":\"Angler Leggings\",\"color\":\"white\",\"italic\":false}]',color:720975,Lore:['[{\"text\":\"Sea Creature Chance: \",\"color\":\"gray\",\"italic\":false},{\"text\":\"+1%\",\"color\":\"red\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"Defense: \",\"color\":\"gray\",\"italic\":false},{\"text\":\"+25\",\"color\":\"green\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"Full Set Bonus: \",\"color\":\"gold\",\"italic\":false},{\"text\":\"Depth Champion\",\"color\":\"gold\",\"italic\":false},{\"text\":\" \",\"color\":\"yellow\",\"italic\":false,\"bold\":true}]','[{\"text\":\"Take \",\"color\":\"gray\",\"italic\":false},{\"text\":\"-30% \",\"color\":\"green\",\"italic\":false},{\"text\":\"damage from Sea Creatures. \",\"color\":\"gray\",\"italic\":false}]','[{\"text\":\"Increase their spawn rate by \",\"color\":\"gray\",\"italic\":false},{\"text\":\"4%\",\"color\":\"yellow\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"Full Set Bonus: \",\"color\":\"gold\",\"italic\":false},{\"text\":\"Deepness Within\",\"color\":\"gold\",\"italic\":false},{\"text\":\" \",\"color\":\"yellow\",\"italic\":false,\"bold\":true}]','[{\"text\":\"Gain \",\"color\":\"gray\",\"italic\":false},{\"text\":\"10❤ \",\"color\":\"red\",\"italic\":false},{\"text\":\"per Fishing level. \",\"color\":\"gray\",\"italic\":false}]','[{\"text\":\"Fishing: \",\"color\":\"gray\",\"italic\":false},{\"text\":\"Level I\",\"color\":\"yellow\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"This item can be reforged!\",\"color\":\"dark_gray\",\"italic\":false}]','[{\"text\":\"COMMON LEGGINGS\",\"color\":\"white\",\"italic\":false,\"bold\":true}]'],color:720975}}"
            },
            {
                "function": "minecraft:set_attributes",
                "modifiers": [
                  {
                    "attribute": "minecraft:generic.armor",
                    "name": "",
                    "amount": 0,
                    "operation": "multiply_total",
                    "slot": ["feet","legs","chest","head"]
                  }
                ]
            }
          ]
        }
      ]
    },
    {
      "rolls": 1,
      "entries": [
        {
          "type": "item",
          "name": "minecraft:leather_boots",
          "functions": [
            {
              "function": "minecraft:set_nbt",
              "tag": "{Base:{Type:\"BOOTS\",Rarity:\"COMMON\",RarityColor:'{\"text\":\"\",\"color\":\"white\"}',AnvilUses:0,RepairCost:0,Enchantments:[],Reforge:\"\",Defense:10,BaseDefense:10,SCChance:1,BaseSCChance:1,Abilities:[{Name:\"Depth Champion\",Activation:\"\",Type:\"Full Set Bonus\"},{Name:\"DeepnessWithin\",Activation:\"\",Type:\"Full Set Bonus\"}]},HideFlags:63,Unbreakable:1b,Description:['[{\"text\":\"Full Set Bonus: \",\"color\":\"gold\",\"italic\":false},{\"text\":\"Depth Champion\",\"color\":\"gold\",\"italic\":false},{\"text\":\" \",\"color\":\"yellow\",\"italic\":false,\"bold\":true}]','[{\"text\":\"Take \",\"color\":\"gray\",\"italic\":false},{\"text\":\"-30% \",\"color\":\"green\",\"italic\":false},{\"text\":\"damage from Sea Creatures. \",\"color\":\"gray\",\"italic\":false}]','[{\"text\":\"Increase their spawn rate by \",\"color\":\"gray\",\"italic\":false},{\"text\":\"4%\",\"color\":\"yellow\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"Full Set Bonus: \",\"color\":\"gold\",\"italic\":false},{\"text\":\"Deepness Within\",\"color\":\"gold\",\"italic\":false},{\"text\":\" \",\"color\":\"yellow\",\"italic\":false,\"bold\":true}]','[{\"text\":\"Gain \",\"color\":\"gray\",\"italic\":false},{\"text\":\"10❤ \",\"color\":\"red\",\"italic\":false},{\"text\":\"per Fishing level. \",\"color\":\"gray\",\"italic\":false}]','[{\"text\":\"Fishing: \",\"color\":\"gray\",\"italic\":false},{\"text\":\"Level I\",\"color\":\"yellow\",\"italic\":false}]','[{\"text\":\"\"}]'],display:{Name:'[{\"text\":\"Angler Boots\",\"color\":\"white\",\"italic\":false}]',color:720975,Lore:['[{\"text\":\"Sea Creature Chance: \",\"color\":\"gray\",\"italic\":false},{\"text\":\"+1%\",\"color\":\"red\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"Defense: \",\"color\":\"gray\",\"italic\":false},{\"text\":\"+10\",\"color\":\"green\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"Full Set Bonus: \",\"color\":\"gold\",\"italic\":false},{\"text\":\"Depth Champion\",\"color\":\"gold\",\"italic\":false},{\"text\":\" \",\"color\":\"yellow\",\"italic\":false,\"bold\":true}]','[{\"text\":\"Take \",\"color\":\"gray\",\"italic\":false},{\"text\":\"-30% \",\"color\":\"green\",\"italic\":false},{\"text\":\"damage from Sea Creatures. \",\"color\":\"gray\",\"italic\":false}]','[{\"text\":\"Increase their spawn rate by \",\"color\":\"gray\",\"italic\":false},{\"text\":\"4%\",\"color\":\"yellow\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"Full Set Bonus: \",\"color\":\"gold\",\"italic\":false},{\"text\":\"Deepness Within\",\"color\":\"gold\",\"italic\":false},{\"text\":\" \",\"color\":\"yellow\",\"italic\":false,\"bold\":true}]','[{\"text\":\"Gain \",\"color\":\"gray\",\"italic\":false},{\"text\":\"10❤ \",\"color\":\"red\",\"italic\":false},{\"text\":\"per Fishing level. \",\"color\":\"gray\",\"italic\":false}]','[{\"text\":\"Fishing: \",\"color\":\"gray\",\"italic\":false},{\"text\":\"Level I\",\"color\":\"yellow\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"This item can be reforged!\",\"color\":\"dark_gray\",\"italic\":false}]','[{\"text\":\"COMMON BOOTS\",\"color\":\"white\",\"italic\":false,\"bold\":true}]'],color:720975}}"
            },
            {
                "function": "minecraft:set_attributes",
                "modifiers": [
                  {
                    "attribute": "minecraft:generic.armor",
                    "name": "",
                    "amount": 0,
                    "operation": "multiply_total",
                    "slot": ["feet","legs","chest","head"]
                  }
                ]
            }
          ]
        }
      ]
    }
  ]
}