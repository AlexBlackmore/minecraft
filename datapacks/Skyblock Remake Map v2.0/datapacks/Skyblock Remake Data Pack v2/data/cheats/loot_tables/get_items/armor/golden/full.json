{
  "pools": [
    {
      "rolls": 1,
      "entries": [
        {
          "type": "item",
          "name": "minecraft:golden_helmet",
          "functions": [
            {
              "function": "minecraft:set_nbt",
              "tag": "{Base:{Vanilla:1b,Type:\"HELMET\",Rarity:\"COMMON\",RarityColor:'{\"text\":\"\",\"color\":\"white\"}',AnvilUses:0,RepairCost:0,Enchantments:[],Reforge:\"\",Defense:10,BaseDefense:10},HideFlags:63,Unbreakable:1b,display:{Name:'[{\"text\":\"Golden Helmet\",\"color\":\"white\",\"italic\":false}]',Lore:['[{\"text\":\"Defense: \",\"color\":\"gray\",\"italic\":false},{\"text\":\"+10\",\"color\":\"green\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"This item can be reforged!\",\"color\":\"dark_gray\",\"italic\":false}]','[{\"text\":\"COMMON HELMET\",\"color\":\"white\",\"italic\":false,\"bold\":true}]']}}"
            },
            {
                "function": "minecraft:set_attributes",
                "modifiers": [
                  {
                    "attribute": "minecraft:generic.armor",
                    "name": "",
                    "amount": 0,
                    "operation": "multiply_total",
                    "slot": ["feet","legs","chest","head"]
                  }
                ]
            }
          ]
        }
      ]
    },
    {
      "rolls": 1,
      "entries": [
        {
          "type": "item",
          "name": "minecraft:golden_chestplate",
          "functions": [
            {
              "function": "minecraft:set_nbt",
              "tag": "{Base:{Vanilla:1b,Type:\"CHESTPLATE\",Rarity:\"COMMON\",RarityColor:'{\"text\":\"\",\"color\":\"white\"}',AnvilUses:0,RepairCost:0,Enchantments:[],Reforge:\"\",Defense:25,BaseDefense:25},HideFlags:63,Unbreakable:1b,display:{Name:'[{\"text\":\"Golden Chestplate\",\"color\":\"white\",\"italic\":false}]',Lore:['[{\"text\":\"Defense: \",\"color\":\"gray\",\"italic\":false},{\"text\":\"+25\",\"color\":\"green\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"This item can be reforged!\",\"color\":\"dark_gray\",\"italic\":false}]','[{\"text\":\"COMMON CHESTPLATE\",\"color\":\"white\",\"italic\":false,\"bold\":true}]']}}"
            },
            {
                "function": "minecraft:set_attributes",
                "modifiers": [
                  {
                    "attribute": "minecraft:generic.armor",
                    "name": "",
                    "amount": 0,
                    "operation": "multiply_total",
                    "slot": ["feet","legs","chest","head"]
                  }
                ]
            }
          ]
        }
      ]
    },
    {
      "rolls": 1,
      "entries": [
        {
          "type": "item",
          "name": "minecraft:golden_leggings",
          "functions": [
            {
              "function": "minecraft:set_nbt",
              "tag": "{Base:{Vanilla:1b,Type:\"LEGGINGS\",Rarity:\"COMMON\",RarityColor:'{\"text\":\"\",\"color\":\"white\"}',AnvilUses:0,RepairCost:0,Enchantments:[],Reforge:\"\",Defense:15,BaseDefense:15},HideFlags:63,Unbreakable:1b,display:{Name:'[{\"text\":\"Golden Leggings\",\"color\":\"white\",\"italic\":false}]',Lore:['[{\"text\":\"Defense: \",\"color\":\"gray\",\"italic\":false},{\"text\":\"+15\",\"color\":\"green\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"This item can be reforged!\",\"color\":\"dark_gray\",\"italic\":false}]','[{\"text\":\"COMMON LEGGINGS\",\"color\":\"white\",\"italic\":false,\"bold\":true}]']}}"
            },
            {
                "function": "minecraft:set_attributes",
                "modifiers": [
                  {
                    "attribute": "minecraft:generic.armor",
                    "name": "",
                    "amount": 0,
                    "operation": "multiply_total",
                    "slot": ["feet","legs","chest","head"]
                  }
                ]
            }
          ]
        }
      ]
    },
    {
      "rolls": 1,
      "entries": [
        {
          "type": "item",
          "name": "minecraft:golden_boots",
          "functions": [
            {
              "function": "minecraft:set_nbt",
              "tag": "{Base:{Vanilla:1b,Type:\"BOOTS\",Rarity:\"COMMON\",RarityColor:'{\"text\":\"\",\"color\":\"white\"}',AnvilUses:0,RepairCost:0,Enchantments:[],Reforge:\"\",Defense:5,BaseDefense:5},HideFlags:63,Unbreakable:1b,display:{Name:'[{\"text\":\"Golden Boots\",\"color\":\"white\",\"italic\":false}]',Lore:['[{\"text\":\"Defense: \",\"color\":\"gray\",\"italic\":false},{\"text\":\"+5\",\"color\":\"green\",\"italic\":false}]','[{\"text\":\"\"}]','[{\"text\":\"This item can be reforged!\",\"color\":\"dark_gray\",\"italic\":false}]','[{\"text\":\"COMMON BOOTS\",\"color\":\"white\",\"italic\":false,\"bold\":true}]']}}"
            },
            {
                "function": "minecraft:set_attributes",
                "modifiers": [
                  {
                    "attribute": "minecraft:generic.armor",
                    "name": "",
                    "amount": 0,
                    "operation": "multiply_total",
                    "slot": ["feet","legs","chest","head"]
                  }
                ]
            }
          ]
        }
      ]
    }
  ]
}